sci-libs/shapelib/files/0018-DBFWriteAttribute-return-true-when-no-precision-loss.patch

```diff
@@ -0,0 +1,36 @@
+From 1a344bfec194b73139f0316167ba27f52ad5ce69 Mon Sep 17 00:00:00 2001
+From: Even Rouault <even.rouault@spatialys.com>
```

Comment:
I think this one is OK to keep as the patch is small and it fixes a problem that looks annoying.